### PR TITLE
POL-223 - RDS Instance Report

### PR DESCRIPTION
**Pull request by srpomeroy**
### Description

Provides a list of DB Instances, the engine type, engine version, and quantity of vcpus.

### Issues Resolved

n/a

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/rds_instance_report/rds_instance_report.pt [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_rds_instances" do
+  validate $ds_filtered_results do
+    summary_template "AWS RDS Instances"
+    detail_template <<-EOS
+# AWS RDS Instances
+| Region | Instance ID | Engine | Engine Version | Instance Class | Availability Zone | vCPUs | License Model |
```
Comment (cdwilhelm) [inline cost/aws/rds_instance_report/rds_instance_report.pt]: The data should be sorted, probably all fields except instance id

File: cost/aws/rds_instance_report/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# AWS RDS Instance Report
```
Comment (cdwilhelm) [inline cost/aws/rds_instance_report/README.md]: Remove the word Report.

File: README.md [hunk 3]
```diff
@@ -45,6 +45,7 @@ Please contact sales@flexera.com to learn more.
 
 - [AWS Unused RDS Instances](./cost/aws/unused_rds/)
 - [AWS RDS Instances RightSizing](./cost/aws/rds_instance_cloudwatch_utilization)
+- [AWS RDS Instances Report](./cost/aws/rds_instance_report)
```
Comment (cdwilhelm) [inline README.md]: Remove the word Report.

File: cost/aws/rds_instance_report/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## What it does
+
+This Policy Template gathers AWS RDS Instance data.
```
Comment (cdwilhelm) [inline cost/aws/rds_instance_report/README.md]: consider what the value of the data is, for licensing etc.

**Comment by cdwilhelm** [comment]
@srpomeroy howdy, are you still working on this?

File: cost/aws/rds_instance_report/rds_instance_report.pt [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+type "policy"
+short_description "Collects all RDS instances in an account. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/aws/rds_instance_report/) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
+severity "low"
+category "Cost"
+info(
+  version: "2.0",
+  provider: "AWS",
+  service: "RDS",
+  policy_set: "RightSize Database Services"
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply cost/aws/rds_instance_report/rds_instance_report.pt]: Thoughts on a policy set? Or should I just leave it null?

**Pull request merged**